svelte/kredeum-create-nft.svelte

```diff
@@ -170,7 +171,14 @@
           </li>
         </ul>
       {:else}
-        <div class="section">
+      <div class="section">
+        <span class="label label-big">NFT title</span>
+        <div class="form-field">
+          <input type="text" placeholder="My NFT title" bind:value={nftTitle}  id="title-nft" />
```

Comment:
mettre par défaut l' imagename (si la photo est chargée) dans le champ de saisie plutôt que 'My NFT title'
aussi plutôt le mettre après l'upload 

i.e. qu'on soit pas obligé d'en définir un, si le nom du fichier est ok, ou simplement de le modifier

Reply:
je ne pense pas que cela soit nécessaire > nous devons pas écraser le titre déjà écrit par l'utilisateur après upload d'un fichier
en revanche j'ai ajouté un message d'erreur si l'utilisateur clique sur "Mint" sans avoir préalablement écrit un titre et uploadé un fichier

Reply:
s'il y a pas de titre après l'upload du fichier, il faut prendre le nom du ficher  (pas l'écraser s'il existe !)  en laissant la possibilité de le modifier ou non avant Mint

parcequ'une fois sur deux l'utilisateur reprendra le nom du fichier, éventuellement en modifiant la casse ou les espaces

ce qui est discutable oui, c de savoir si l'on met l'upload de photo avant la saisie du titre ou après

Reply:
ca rend le Mint moins simple, en rajoutant une étape obligatoire, qui devrait rester en option

"The Easiest way to Mint"